### PR TITLE
Update intro font sizes and rationalise intro styles

### PR DESCRIPTION
**Pull request by helenb**
[Link to Ticket](https://torchbox.monday.com/boards/1192293412/pulses/1579462155) - but see in particular [this update](https://torchbox.monday.com/boards/1192293412/pulses/1579462155/posts/311927942) as the brief changes

### Description of Changes Made

- Rationalise all intro block classes
- Remove the heading classes on intro blocks and give them their own custom font-size, which needs to override the rich text font-size
- Adjust the margins around the work page intro at mobile
- Show the intro field in the pattern library for blog and standard pages

### How to Test

http://localhost:8000/pattern-library/pattern/patterns/pages/standard/standard_page.html
http://localhost:8000/pattern-library/pattern/patterns/pages/blog/blog_detail.html
http://localhost:8000/pattern-library/pattern/patterns/pages/service/service_page.html
http://localhost:8000/pattern-library/pattern/patterns/pages/work/work_page.html

### Screenshots

<details>
  <summary>Expand to see more</summary>

Blog page

Screenshot 2024-07-30 at 15 33 39

Standard page

Screenshot 2024-07-30 at 15 33 18

Work page (mobile)

Screenshot 2024-07-30 at 15 32 28

Work page (desktop)

Screenshot 2024-07-30 at 15 32 58

Service page

Screenshot 2024-07-30 at 15 34 12

</details>

### MR Checklist

- [x] Add a description of your pull request and instructions for the reviewer to verify your work.
- [x] If your pull request is for a specific ticket, link to it in the description.
- [x] Stay on point and keep it small so the merge request can be easily reviewed.
- [x] Tests and linting passes.

#### Unit tests

- [ ] Added
- [x] Not required

#### Documentation

- [ ] Updated build docs
- [ ] Updated editor guidelines (See https://intranet.torchbox.com/torchbox-com-project-docs for the private link, for Torchbox employees only)
- [ ] Updated field spec (See https://intranet.torchbox.com/torchbox-com-project-docs for the private link, for Torchbox employees only)
- [x] Not required

#### Browser testing

- [x] I have tested in the following browsers and environments (edit the list as required)
  - Latest version of Chrome on mac
- [ ] Not required

#### Data protection

- [x] Not relevant
- [ ] This adds new sources of PII and documents it and modifies Birdbath processors accordingly

#### Accessibility

- [x] Automated WCAG 2.1 tests pass
- [x] HTML validation passes
- [ ] Manual WCAG 2.1 tests completed
- [ ] I have tested in a screen reader
- [ ] I have tested in high-contrast mode
- [ ] Any animations removed for prefers-reduced-motion
- [ ] Not required

#### Sustainability

- [ ] Images are optimised and lazy-loading used where appropriate
- [ ] SVGs have been optimised
- [ ] Perfomance and transfer of data considered
- [ ] If JavaScript is needed alternatives have been considered
- [x] Not required

#### Pattern library

- [x] The pattern library component for this template displays correctly, and does not break parent templates
- [ ] The styleguide is updated if relevant
- [ ] Changes are not relevant the pattern library

### CODE REVIEW

File: tbx/project_styleguide/templates/patterns/pages/service/service_page.html [hunk 1]
```diff
@@ -8,8 +8,8 @@
         </div>
 
         {% if page.intro %}
-            <div class="grid__intro streamfield__intro">
-                <div class="heading heading--four">{{page.intro|richtext}}</div>
```
Comment (chris-lawton) [inline tbx/project_styleguide/templates/patterns/pages/service/service_page.html:12]: Suggestion: Removing the `heading` class has changed the colour and the font weight, I think they should be `white` and `300`. Here's prod vs local:

<details><summary>Details</summary>
<p>

image


</p>
</details>

File: tbx/project_styleguide/templates/patterns/pages/team/team_detail.html [hunk 2]
```diff
@@ -21,8 +21,8 @@
         </div>
 
         {% if page.intro %}
-            <div class="grid__intro streamfield__intro">
-                <div class="heading heading--four">{{page.intro|richtext}}</div>
+            <div class="grid__intro streamfield__intro rich-text">
```
Comment (chris-lawton) [inline tbx/project_styleguide/templates/patterns/pages/team/team_detail.html:24]: Suggestion: Could this and the other instances in `work_page.html`, `service_page.html` use an include to the `intro_block.html`? E.g.

```
{% include "patterns/molecules/streamfield/blocks/intro_block.html" with value=page.intro %}
```

File: tbx/project_styleguide/templates/patterns/pages/service/service_page.html (hunk 1 above)
Comment (chris-lawton) [reply tbx/project_styleguide/templates/patterns/pages/service/service_page.html:12]: This also needs fixing on the Work page

File: tbx/project_styleguide/templates/patterns/pages/team/team_detail.html (hunk 2 above)
Comment (helenb) [reply tbx/project_styleguide/templates/patterns/pages/team/team_detail.html:24]: ah interesting - will have a think.

File: tbx/project_styleguide/templates/patterns/pages/service/service_page.html (hunk 1 above)
Comment (helenb) [reply tbx/project_styleguide/templates/patterns/pages/service/service_page.html:12]: that should now be fixed.

File: tbx/project_styleguide/templates/patterns/pages/team/team_detail.html (hunk 2 above)
Comment (helenb) [reply tbx/project_styleguide/templates/patterns/pages/team/team_detail.html]: I've created a new common intro file at `molecules/intro/intro.html` - this is then included by all other places where we use an intro, including the intro streamfield block. Think that's slightly cleaner than including the block template directly.

File: tbx/project_styleguide/templates/patterns/pages/service/service_page.html (hunk 1 above)
Comment (chris-lawton) [reply tbx/project_styleguide/templates/patterns/pages/service/service_page.html:12]: Looks good!

**Review by chris-lawton — APPROVED** [review]

**Pull request merged**